kit/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Thanks for adding this. Let's do a single LICENSE file at the top of the repo. I want to avoid nesting licenses as it may make folks wary of licenses they haven't found yet in the tree.

Reply:
My understanding is that this is not a permanent place holder for the whole KIT Eco system project to co-exist in one repository. So, it will be easier to migrate to other repositories with folders containing all of it together than picking and choosing certain files from parent folder when we move to individual repositories ?  wdyt ?

Reply:
Agree in general, minus the LICENSE file.